squire_lib/src/accounts.rs

```diff
@@ -9,6 +10,22 @@ use crate::{
     tournament_manager::TournamentManager,
 };
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub enum SharingPermissions {
+    Everything,
+    OnlyDeckList,
+    OnlyDeckName,
+    Nothing
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub enum Platforms {
```

Comment:
It is probably a good idea to have an `Other` or `Custom` variant. This variant should contain a string. In theory, a user could have many of these custom tags.

Reply:
Alright that makes sense